api/src/lib/aws.ts

```diff
@@ -117,8 +119,12 @@ export async function s3PutSignedUrl(
 }
 
 export async function deleteUploadFile(upload: Upload) {
-  const fileKey = `uploads/${upload.agency.organizationId}/${upload.agencyId}/${upload.reportingPeriodId}/${upload.id}/${upload.filename}`
-  await s3DeleteObject(fileKey)
+  try {
+    const fileKey = `uploads/${upload.agency.organizationId}/${upload.agencyId}/${upload.reportingPeriodId}/${upload.id}/${upload.filename}`
+    await s3DeleteObject(fileKey)
+  } catch (error) {
+    logger.error(error, 'Error deleting file from S3:')
+  }
```

Comment:
I'm curious about why we're suppressing this error instead of letting it bubble up or re-throwing it after logging. I would think a hard error would make it more apparent that there was a permissions issue to address in the first place, but maybe I'm missing something.

Reply:
that's a good point. let me remove the catch block for now. Since this function is only being executed by the script bubbling up the error would be ideal functionality